CMakeLists.txt

```diff
@@ -16,9 +16,16 @@ set(cdat_CMAKE_SOURCE_DIR ${cdat_SOURCE_DIR}/CMake)
 set(cdat_CMAKE_BINARY_DIR ${cdat_BINARY_DIR}/CMake)
 set(cdat_external_patch_dir ${cdat_SOURCE_DIR}/exsrc)
 
-# Helper variables to locate programs and libraries
-set(SB_LIB_DIR "${CMAKE_INSTALL_PREFIX}/lib")
-set(SB_EXTERNALS_DIR "${CMAKE_INSTALL_PREFIX}/Externals")
+if (APPLE)
+  set(SB_EXTERNALS_DIR "${CMAKE_INSTALL_PREFIX}/Externals")
+  set(SB_DIR "${CMAKE_INSTALL_PREFIX}/Library/Frameworks/Python.framework/Versions/2.7")
```

Comment:
Alright, I'm testing a build using the ${PYVER} variable, though I had to move this if block down to just above the includes, and I include python prior to it (so I have the version #)